anger.js

```diff
@@ -3,19 +3,29 @@
 const EE = require('events').EventEmitter
 const nes = require('nes')
 const Histogram = require('native-hdr-histogram')
+const tryAgain = require('try-again')
 const histUtil = require('hdr-histogram-percentiles-obj')
 const steed = require('steed')
 const get = require('lodash.get')
 const Client = nes.Client
+const again = tryAgain({
+  retries: 8,
+  max: 10000,
+  jitter: 0.2,
+  factor: 2,
+  min: 100
+})
```

Comment:
I think this is instance specific, should it be inside `anger()`?

I would make that configurable, so we can tune down the `max` and reduce the timeout for the test.
What do you think?